autoload/lsp/lsp.vim

```diff
@@ -41,9 +41,9 @@ def LspInitOnce()
 		{name: 'LspDiagHint', text: 'H>', texthl: 'Question',
 						linehl: 'MatchParen'}])
 
-  prop_type_add('LspTextRef', {highlight: 'Search'})
-  prop_type_add('LspReadRef', {highlight: 'DiffChange'})
-  prop_type_add('LspWriteRef', {highlight: 'DiffDelete'})
+  prop_type_add('LspTextRef', {highlight: 'ErrorMsg', override: true})
```

Comment:
Why is the highlight group changed from 'Search' to 'ErrorMsg'?  The ErrorMsg highlight group uses a red background.  In this case as there is no error, the Search highlight group (with a yellow background) was used.